### ISSUE TITLE
Midterm Peer Review - atp44

### ISSUE THREAD

**Issue by anthon11786** [meredithdobrzynski/google_play_project#9]
I was impressed by the professional look and style of this report. It has very organized sections, clear visualizations, and is written clearly and straightforward. I thought the layout of the different types of data you’re dealing with was very helpful to the reader, especially you’re breakdown of how you handled preprocessing each of your features. I thought the visualizations were very insightful and well presented (nice labeling and style). Also the clear discussion of the next steps, i.e. discussing the possible loss functions, regularizers, and using k-cross-fold validation gives the reader a great idea of what steps you are going to take next in your analysis. 

I thought it would be helpful to include how many missing values there were in your target column, Rating. Since you are imputing the mean into the target column I think it would be important to include what percent of the rows you had to impute. Also, when discussing the feature LastUpdated, I think that there is an argument for keeping the feature because it shows an active developer community backing the app, i.e. if users are reporting issues and the app hasn’t been updated since last year they know it won’t be fixed and may stop using the app.  Finally, I think you can address how you plan to generalize well, i.e. avoiding overfitting/underfitting, and possibly discuss if you think if your current model is avoiding it already.
